src/main/java/com/mathworks/ci/freestyle/RunMatlabCommandBuilder.java

```diff
@@ -110,6 +114,14 @@ public boolean isApplicable(
                 @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobtype) {
             return true;
         }
+        @POST

```

Comment:
I don't think that this is necessary. Also, we should add a blank line above it for whitespace consistentcy.

Reply:
This Recommended approach for avoiding CSRF attacks please refer [this](https://www.jenkins.io/doc/developer/security/form-validation/#protecting-from-csrf)  also we had received the Jenkins issue from Jenkins maintainers earlier to use this approach from security perspective